### PR TITLE
[DEBUG][CI]travis: add make deb check

### PR DESCRIPTION
**Pull request by xiulipan**
add make bindeb-pkg check to travis-ci

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>

### CODE REVIEW

**Comment by xiulipan** [comment]
@plbossart @lgirdwood 
If this would be ok for you about #402

File: .travis.yml [hunk 1]
```diff
@@ -29,9 +29,11 @@ jobs:
         - make defconfig
         - scripts/kconfig/merge_config.sh .config kconfig/base-defconfig kconfig/sof-defconfig
         - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
+        - make bindeb-pkg -j`getconf _NPROCESSORS_ONLN`
```
Comment (plbossart) [inline .travis.yml:32]: wondering if you can leave the regular make command and do the bindeb-pkg once the kernel actually builds. The Debian stuff can be slow, you'd want to compile first without it.

File: .travis.yml [hunk 2]
```diff
@@ -29,9 +29,11 @@ jobs:
...
     - name: "BUILD SST Kernel x86_64"
       script:
         - export ARCH=x86_64
         - make defconfig
         - scripts/kconfig/merge_config.sh .config kconfig/base-defconfig kconfig/sst-defconfig
         - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
+        - make bindeb-pkg -j`getconf _NPROCESSORS_ONLN`
```
Comment (plbossart) [inline .travis.yml]: this one we don't care abou.t

File: .travis.yml (hunk 1 above)
Comment (xiulipan) [reply .travis.yml:32]: @plbossart 
```
        - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
        - make bindeb-pkg -j`getconf _NPROCESSORS_ONLN`
```

The first line is the original make. Then followed the deb build

File: .travis.yml (hunk 2 above)
Comment (xiulipan) [reply .travis.yml]: ok, will remove.

File: .travis.yml (hunk 1 above)
Comment (plbossart) [reply .travis.yml:32]: Ah yes, sorry. that's fine then

**Pull request merged**